src/js/controller/tracks-helper.ts

```diff
@@ -1,4 +1,18 @@
-export function createId(track, tracksCount) {
+import type { TextTrackLike } from 'types/generic.type';
+import type { PlaylistItemTrack } from 'playlist/track';
+
+export type Track = (TextTrackLike | PlaylistItemTrack) & {
```

Comment:
`createLabel` is only used (ignore test) in captions.js. Maybe that's where we should define `Track` or `CaptionsTrack`? Maybe we should just move `createLabel` there too.

Reply:
Good call. Removing the exports. We can move it over when caption.js is typescripted.